lxd/instance/drivers/driver_qemu.go

```diff
@@ -2897,7 +2897,16 @@ func (d *qemu) generateQemuConfigFile(cpuInfo *cpuTopology, mountInfo *storagePo
 		cfg = append(cfg, qemuUSB(&usbOpts)...)
 	}
 
-	devBus, devAddr, multi = bus.allocate(busFunctionGroupNone)
+	if shared.IsTrue(d.expandedConfig["security.csm"]) {
+		// Allocate a regular entry to keep things aligned normally.
+		_, _, _ = bus.allocate(busFunctionGroupNone)
```

Comment:
Does this not get used for anything?

Reply:
Yes, that's to avoid breaking the NIC names primarily. So it's an empty root PCIe port just to keep things at their usual location.